libnavigation-base/src/main/java/com/mapbox/navigation/base/metrics/MetricsReporter.kt

```diff
@@ -0,0 +1,12 @@
+package com.mapbox.navigation.base.metrics
+
+import com.mapbox.android.telemetry.Event
```

Comment:
we should not leak `telemetry.Event` in public interface as it will force our customers to depend on our telem library